modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -82,6 +83,59 @@ object SeqexecEngine {
     willStopIn: Option[Int]
   )
 
+  object SequenceView {
+
+    import edu.gemini.seqexec.engine
+
+    // TODO: Better name and move it to `engine`
+    type QueueAR = engine.Queue[engine.Action \/ engine.Result]
```

Comment:
It's a poor mnemonic for `Action/Result` but we have to rename it. I already opened a ticket to do a general cohesive renaming because this is just one case of several poor naming choices.